torchvision/prototype/features/_pixels.py

```diff
@@ -0,0 +1,5 @@
+from ._feature import Feature
+
+
+class Pixels(Feature):
+    pass
```

Comment:
What are the semantics that we envision for `Pixels`? What is its format structure? Can it be tied to `Keypoint`s?

Reply:
> What are the semantics that we envision for `Pixels`?

I don't have anything specific in mind yet, besides that `data.shape == (N, 2)`.

> What is its format structure?

Could you elaborate what you mean?

> Can it be tied to `Keypoint`s?

Yes. `KeyPoints` would be a good subclass of `Pixels`.

Reply:
Failing tests are related to this. I'll fix them after we decide what to do here. I'm also fine with returning this as a vanilla tensor for now until we have a `Contour` feature.

Reply:
I think I would encourage returning a vanilla tensor for now, until we have a good semantic for what it should be.

One thing I would like to avoid is proliferation of custom types without clear semantics, so I would like to treat each type with utmost care and consideration